EternalBAND.Api/Options/JwtTokenOptions.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace EternalBAND.Api.Options
+{
+    public class JwtTokenOptions
+    {
+        public const string JwtOptionKey = "JwtTokenSettings";
+
+        public string Secret { get; set; } = "";
```

Comment:
public string Secret { get; set; } = "";

Let's talk later about where this private key will be stored.

Reply:
there is an option to hold kind of secrets in .net name is .net secret manager
https://learn.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-8.0&tabs=windows

Of course we can implement other logics if you have any suggestions